feeds/wlan-ap/opensync/files/bin/auto-conf

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# All this script does at the moment is find the highest mode of operation
```

Comment:
These comments are overkill.  They are longer than the code.

Reply:
I guess I'm a fan of well documented code haha. In the future should I only add comments where it's needed to explain what the code does? I've always been taught the more comments the better when working on large code projects to help maintainability. What's a good rule to follow for this kind of stuff?